From d1d40cda51078a5dd8843d8c54e7e112fa028518 Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Mon, 29 Feb 2016 16:47:18 -0800
Subject: [PATCH 34/36] docs: Add list of filesystems for fs-type (#1311596)

Add the most common filesystem types to the parted manpage.

Resolves: rhbz#1311596
(cherry picked from commit c9f50a36aee0a4c373b5e1bd3069aeb18fac055d)
---
 doc/C/parted.8 | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/doc/C/parted.8 b/doc/C/parted.8
index 5304375..a45f42f 100644
--- a/doc/C/parted.8
+++ b/doc/C/parted.8
@@ -76,6 +76,8 @@ should be one of "aix", "amiga", "bsd", "dvh", "gpt", "loop", "mac", "msdos",
 .B mkpart \fIpart-type\fP \fI[fs-type]\fP \fIstart\fP \fIend\fP
 Make a \fIpart-type\fP partition for filesystem \fIfs-type\fP (if specified),
 beginning at \fIstart\fP and ending at \fIend\fP (by default in megabytes).
+\fIfs-type\fP can be one of "btrfs", "ext2", "ext3", "ext4", "fat16", "fat32",
+"hfs", "hfs+", "linux-swap", "ntfs", "reiserfs", or "xfs".
 \fIpart-type\fP should be one of "primary", "logical", or "extended".
 .TP
 .B name \fIpartition\fP \fIname\fP
-- 
2.5.5

